neural_network_regression/nn_regression.cpp

```diff
@@ -14,18 +15,16 @@
 
 /**
  * ABOUT THE DATASET bodyfat.tsv (freely available dataset).
- * 
- * The bodyfat dataset contains estimates of the percentage of body fat determined by
- * underwater weighing and various body circumference measurements for 252 men.
- * Accurate measurement of body fat is very expensive,but by using machine learning
- * it is possible to calculate a prediction with good accuracy by just using some low cost
- * measurements of the body. The columns in the dataset are the following:
- * 
- * Percent body fat (%) => this is the decision column (what we want to get from the model).
- * Age (years)
- * Weight (lbs)
- * Height (inches)
- * Neck circumference (cm)
+ *
+ * The bodyfat dataset contains estimates of the percentage of body fat
+ * determined by underwater weighing and various body circumference measurements
+ * for 252 men. Accurate measurement of body fat is very expensive,but by using
```

Comment:
```suggestion
 * for 252 men. Accurate measurement of body fat is very expensive, but by using
```
Just noticed this while passing through. :)